Shared/Samples/Create load report/CreateLoadReportView.Model.swift

```diff
@@ -0,0 +1,299 @@
+                traceParameters.traceConfiguration?.traversability?.barriers = UtilityTraceOrCondition(
+                    leftExpression: initialExpression,
+                    rightExpression: phasesAttributeComparison
+                )
+                
+                let traceResults = try await utilityNetwork.trace(using: traceParameters)
+                
+                var totalCustomers = 0
+                var totalLoad = 0
+                
+                for result in traceResults {
+                    switch result {
+                    case let elementResult as UtilityElementTraceResult:
+                        // Get the unique customers count.
+                        totalCustomers = Set(elementResult.elements.map(\.objectID)).count
+                    case let functionResult as UtilityFunctionTraceResult:
+                        // Get the total load with a function output.
+                        totalLoad = Int(functionResult.functionOutputs[1].result as! Double)
+                    default:
+                        break
+                    }
+                }
+                summaries[phase] = PhaseSummary(
+                    totalCustomers: totalCustomers,
+                    totalLoad: totalLoad
+                )
+            }
+            resetEnabled = true
+            statusText = nil
+        }
+        
+        func addPhase(_ phase: CodedValue) {
+            includedPhases.append(phase)
+            excludedPhases = excludedPhases.filter { $0 != phase }
+            sortPhases()
+            runEnabled = true
+        }
+        
+        func deletePhase(_ phase: CodedValue) {
+            excludedPhases.append(phase)
+            includedPhases = includedPhases.filter { $0 != phase }
+            sortPhases()
+            summaries[phase] = nil
+        }
+        
+        func deletePhase(atOffsets indexSet: IndexSet) {
+            includedPhases.remove(atOffsets: indexSet)
+            let union = Set(includedPhases).union(Set(excludedPhases))
+            let subtraction = Set(allPhases).subtracting(Array(union))
+            if let phase = subtraction.first {
+                deletePhase(phase)
+            }
+        }
+        
+        func sortPhases() {
+            includedPhases = includedPhases.sorted { $0.name < $1.name }
+            excludedPhases = excludedPhases.sorted { $0.name < $1.name }
+        }
+        
+        func summaryForPhase(_ phase: CodedValue) -> String? {
+            if let summary = summaries[phase] {
+                let formattedString = String(
+                    format: "C: %@  L: %@",
+                    numberFormatter.string(from: summary.totalCustomers as NSNumber)!,
+                    numberFormatter.string(from: summary.totalLoad as NSNumber)!
+                )
+                return formattedString
+            } else {
+                return nil
+            }
+        }
```

Comment:
I think `String(format:_)` is not what we should use in SwiftUI.
@philium is using interpolated string the recommended way?
Such as `"C: \(summary.totalCustomers.formatted()), L: \(summary.totalLoad.formatted())"` (with the right format style)

Reply:
Yes, that works. Or, if you want more control over the format, use string interpolation of the value itself and provide a format style to use. Either way, this method (and anything format-related for display to the user) should go on the view itself:

```swift
func summaryForPhase(_ phase: CodedValue) -> LocalizedStringKey {
    let format: IntegerFormatStyle = .number
    if let summary = model.summaries[phase] {
        return "C: \(summary.totalCustomers, format: format)  L: \(summary.totalLoad, format: format)"
    } else {
        return "N/A"
    }
}
```

Reply:
Thank you, I updated the samples contribution guideline with these localization considerations.